decorator/decorator.rb

```diff
@@ -0,0 +1,24 @@
+require '../classes/nameable'
+
```

Comment:
- [ ] The same problem is here, Please fix this issue. You can use `require './classes/nameable'` or  `require_relative '../classes/nameable'`